test-coverage.txt

```diff
@@ -0,0 +1,43 @@
+ PASS  src/components/Header/DarkmodeButton/DarkmodeButton.test.js
+ PASS  src/components/Header/ProfileButton/ProfileButton.test.js
+ PASS  src/components/Header/MenuButton/MenuButton.test.js
+ PASS  src/components/Header/Searchbar/Searchbar.test.js
+ PASS  src/components/VideosGrid/VideoGridItem/VideoGridItem.test.js
+ PASS  src/components/Header/Header.test.js
+ PASS  src/components/VideosGrid/VideoGrid.test.js
+ PASS  src/components/Layout/Layout.test.js
+ PASS  src/components/App/App.test.js
+-------------------------------------|----------|----------|----------|----------|-------------------|
+File                                 |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+-------------------------------------|----------|----------|----------|----------|-------------------|
+All files                            |    93.18 |       50 |    92.31 |    93.18 |                   |
```

Comment:
Good code coverage, the branch percentage is a little bit low, you can improve this by adding more cases to the tests, for example if you have a something like:

```jsx
if (isOpened) {
  // code when isOpen is true
} else {
  // code when isOpen is false
}
```
the way to test this code and have a 100% coverage is by creating a case that makes `isOpen = true` and another case that makes `isOpen = false`, with this both parts of the `if` statement will be executed and the branch percentage will increase